modules/gapi/test/gapi_fluid_test_kernels.hpp

```diff
@@ -112,6 +113,13 @@ G_TYPED_KERNEL_M(TSplit3_4lpi, <GMat3(GMat)>, "test.fluid.split3_4lpi") {
     }
 };
 
+G_TYPED_KERNEL(TSumArrayToMat, <GMat(GArray<cv::Mat>)>, "test.fluid.sum_array_to_mat")
+{
+    static GMatDesc outMeta(const cv::GArrayDesc&) {
+        return GMatDesc(CV_8U, 1, cv::Size(3, 3));
```

Comment:
Since we have no working GArrayDesc, there is no other ways to fill the Desc

Reply:
I'd avoid testing cases like this.

Let's make `equalizeHistogram` kernel with GArray<T> input argument (a LUT). If it doesn't require more efforts

Reply:
Done